docs/en/Getting-Started-Create-Solution-Single.md

```diff
@@ -0,0 +1,52 @@
+# Getting Started
+
+````json
+//[doc-params]
+{
+    "UI": ["MVC", "Blazor", "BlazorServer", "NG"],
+    "DB": ["EF", "Mongo"]
+}
+````
+
+> This document assumes that you prefer to use **{{ UI_Value }}** as the UI framework and **{{ DB_Value }}** as the database provider. For other options, please change the preference on top of this document.
+
+## Create a New Project
+
+We will use the ABP CLI to create a new ABP project.
+
+> Alternatively, you can **create and download** projects from the [ABP Framework website](https://abp.io/get-started) by easily selecting all options from the page.
```

Comment:
I think this can be misleading because on the get-started page, only the CLI command is generated according to the options. (the project is not generated)